packages/jest-each/src/bind.js

```diff
@@ -23,12 +23,31 @@ const SUPPORTED_PLACEHOLDERS = /%[sdifjoOp%]/g;
 const PRETTY_PLACEHOLDER = '%p';
 const INDEX_PLACEHOLDER = '%#';
 
+const errorWithStack = (message, callsite) => {
```

Comment:
could possibly stick this in jest-util or something, we do it all over the place

Reply:
👍 I'll make that refactor in a separate PR